container/build.py

```diff
@@ -240,6 +286,8 @@ def parse_arguments():
 
   optional.add_argument(
       "-h", "--help", help="print this help text and exit", action="help")
+  optional.add_argument(
+      "-m", "--map", help="overrides target map file path", type=str, default=None)
```

Comment:
be specific this is a path, and relative or absolute or either

Reply:
Mentioned it can be either.